neural_speed/__init__.py

```diff
@@ -189,7 +188,7 @@ def generate(self, input_ids, streamer=None, interactive=False, ignore_prompt=Fa
             if stopping_criteria is not None:
                 if stopping_criteria(torch.tensor(ret), None):
                     break
-            elif ret[0][-1] == self.tokenizer.eos_token_id or \
+            elif ret[0][-1] == self.__get_eos_id() or \
```

Comment:
why change it? Should they be the same? Or do you want to remove the `tokenizer` dependency？

Reply:
we want load bin/gguf, so we need remove the tokenizer dependency,